### DISCUSSION TITLE
What’s the correct ES6 import?

### DISCUSSION THREAD

**Discussion by dattito** [bergie/passport-saml#661] [Q&A] [answered]
I just want to ask what’s the ES6-based import of „const SamlStrategy = require('passport-saml').Strategy;“?

**Comment by dattito** [comment, marked as answer]
I think that `import { Strategy as SamlStrategy } from "passport-saml;` should work

**Reply by markstos** [reply]
Yes, exactly that type of syntax is found in the test suite, although the test suite of course references its own files:
```
import { Strategy as SamlStrategy } from "../../src/passport-saml";
```

**Comment by cjbarth** [comment]
Are you referencing something inside the `passport-saml` codebase?

**Reply by dattito** [reply]
I want to use passport-saml together with blitz.js, so I wanted to import it in the ES6-Style for consistency.
